src/ui/public/vislib/__tests__/visualizations/column_chart.js

```diff
@@ -69,8 +69,16 @@ dataTypesArray.forEach(function (dataType, i) {
         });
       });
 
-      it('should stack values', function () {
-        expect(isStacked).to.be(true);
+      it('should stack values when mode is stacked', function () {
+        if (mode === 'stacked') {
```

Comment:
We shouldn't be writing tests this way: conditional assertions are bad news because if these tests were to never run, that should be a huge red flag for us, but we would never catch that with these tests.  This all stems from the fact that these tests are all executed in a loop, which we shouldn't be doing either.  Duplication across tests isn't something we should be programming our way around, it just makes us more likely to have bugs in the tests themselves, and it makes them harder to reason about, which is the absolute last thing we want from our tests. Plus, it tends to create problems like this conditional test stuff.

I'm OK with this test for the sake of getting this fix in, but please follow up with a new pull request that splits these tests out of the loop they're in and runs these stack/mode tests where appropriate without conditions.